docs/userguides/compile.md

```diff
@@ -36,6 +36,9 @@ contract.my_method()
 2. **Pre-existing Contract Types**: If you have a contract type JSON that was compiled elsewhere, you can include it in your project.
    This is useful if you are unable or unwilling to install a compiler.
 
+3. **Pre-existing Contract Binary**: If you have an artifact with binary compiled elsewhere, you can include it in your project.
```

Comment:
Can you show how to do this?

Reply:
In any brownie project, if you run `brownie compile` then the resulting .json files in `build/contracts/` have the bytecodes included. the process would be similar for hardhat, and i suspect every other tooling around.

Reply:
I realize now you are looking to add this from raw compilation output. The reason I got confused is because the logic is happening in the `ContractType.json()`-like handler